SearchAPI/maturities.yml

```diff
@@ -86,39 +104,36 @@ prod:
     cmr_health: /search/health
     cmr_api: /search/granules.echo10
     cmr_collections: /search/collections
-    cmr_clear_scroll: /search/clear-scroll
-    cmr_scroll: True
     cmr_page_size: 250
     cmr_headers:
-        Client-Id: searchapi_asf
+        Client-Id: prod_searchapi_asf
```

Comment:
This one should definitely not change. CMR uses these to identify our traffic and changing them would affect collection of those metrics. There's no need to normalize to anything on these, since only the CMR team uses them.